samples/flip-coin/README.md

```diff
@@ -7,7 +7,7 @@ is used to demonstrate the use of conditions.
 
 * [Installing Tekton Pipelines on OpenShift](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#installing-tekton-pipelines-on-openshift)
 
-There are two ways to generate `Tekton` YAML. One is using `dsl-compile-tekton` command line tool, another way is using `TektonCompiler` inside your pipeline script. Both will generate the same `Tekton` YAML file. Then apply the YAML to your cluster by using the `tkn` command.
+There are two ways to generate `Tekton` YAML. One is using `dsl-compile-tekton` command line tool, another way is using `TektonCompiler` inside your pipeline script. Both will generate the same `Tekton` YAML file. Then either run the YAML to your Kubeflow cluster by using the Kubeflow pipeline UI or use the SDK command line.
```

Comment:
Since our one SDK has both the Argo and Tekton compiler, we decided to define a separate command to call the Tekton compiler so that we can add our own flags such as opt-out artifacts for non-KFP use cases.

Reply:
am not sure the nomenclature works here - looking at current SDK

_kfp.compiler includes classes and methods for building Docker container images for your pipeline components. Methods in this package include, but are not limited to, the following:

kfp.compiler.Compiler.compile_ 

Now in our case, at best, it can be something like

kfp-tekton.compiler.Compiler.compile_ 

or better still

kfp.tekton-compiler.Compiler.compile?

Reply:
The `dsl-compile-tekton` is a command line tool for our Tekton compiler. It's derived from the `dsl-compile` command line tool that the original SDK has for Argo.